kubectl.yaml

```diff
@@ -28,7 +28,7 @@ pipeline:
   - uses: fetch
     with:
       uri: https://github.com/kubernetes/kubernetes/archive/v${{package.version}}.tar.gz
-      expected-sha512: 1cbd7b708a70dd51fe399924ea3d995c809e426a2946eaf50eda041ece7a4740513c54ac871270d2f4d7497f59b551a78e5cb6ecb991d4ba038972c7b7383cac
+      expected-sha256: e5573d6f27f6205c0cde87ebd90c8d1424511a2694faf208e04ce9cae8e17ed2
```

Comment:
I think the expected digest is `a0de781db6d09d91f39306740420f150c5912606f52e4c9a859649f4313fe508`.

(This may be the result of a mistake I make all the time, `curl` without `-L`, which means redirects aren't followed and I hash the wrong thing.)

Reply:
Right you are! I was actually taking the digest of https://github.com/kubernetes/kubernetes/releases/v1.26.0/kubernetes.tar.gz, which was just wrong.

Yay for CI!